### PR TITLE
Make componenttests run single threaded for now.

### PR DESCRIPTION
**Pull request by KevinRansom**

### CODE REVIEW

File: tests/FSharp.Compiler.ComponentTests/xunit.runner.json [hunk 1]
```diff
@@ -2,4 +2,6 @@
     "$schema": "https://xunit.net/schema/current/xunit.runner.schema.json",
     "appDomain": "denied",
     "shadowCopy": false
```
Comment (vzarytovskii) [inline tests/FSharp.Compiler.ComponentTests/xunit.runner.json:4]:
```suggestion
    "shadowCopy": false,
```

**Review by vzarytovskii — APPROVED** [review]

**Comment by vzarytovskii** [comment]
Thanks, Kevin!

**Comment by KevinRansom** [comment]
restarted due to: macos build cancelled

**Comment by KevinRansom** [comment]
passed:  gonna close and open again.

**Comment by KevinRansom** [comment]
passed: gonna try again.

**Comment by KevinRansom** [comment]
failure is a known issue and unrelated ... merging.

**Pull request merged**